api/src/main/java/org/pmiops/workbench/db/model/DbUserTermsOfService.java

```diff
@@ -0,0 +1,75 @@
+package org.pmiops.workbench.db.model;
+
+import java.sql.Timestamp;
+import java.util.Objects;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import org.hibernate.annotations.CreationTimestamp;
+
+@Entity
+@Table(name = "user_terms_of_service")
+public class DbUserTermsOfService {
+  private long userTermsOfServiceId;
+  private long userId;
+  private int tosVersion;
+  private Timestamp agreementTime;
+
+  @Id
+  @GeneratedValue(strategy = GenerationType.IDENTITY)
+  @Column(name = "user_terms_of_service_id")
+  public long getUserTermsOfServiceId() {
+    return userTermsOfServiceId;
+  }
+
+  public void setUserTermsOfServiceId(long userTermsOfServiceId) {
+    this.userTermsOfServiceId = userTermsOfServiceId;
+  }
+
+  @Column(name = "user_id", nullable = false)
```

Comment:
Also curious about the nullable annotation. Is it for documentation or does it affect our code somehow?

Reply:
Spring can use this to talk to the database in a more intelligent way. +1 for readable annotations instead of comments; -1 that we have to specify this stuff multiple places.

Reply:
My understanding is that these impact how the H2 database is generated for unit tests. It's slightly more realistic for those tables to match the nullability constraints in our actual Liquibase-defined SQL schema.

(Although, that being said, see a long comment I added to the "agreement_time" column. Due to some weirdness with how the CreationTimestamp annotation is used by Hibernate, I wasn't able to mark this column as non-nullable in the @Column annotation, even though it is non-nullable in SQL.)